### PR TITLE
3.0.0 beta relnotes

### PR DESCRIPTION
**Pull request by yacovm**

### CODE REVIEW

File: release_notes/v3.0.0-beta.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Consider using the BFT orderer if true decentralization is required, where up to and not including a third of the parties running the orderers may not be trusted due to malicious intent or being compromised.
+For more details of the BFT ordering service and other recent features, see the [What's New documentation](https://hyperledger-fabric.readthedocs.io/en/latest/whatsnew.html).
+
+The v3.0.0-beta release differs from the v3.0.0-preview release by making the peer and orderer block replication resistant to block withholding. 
+Block withholding is when an orderer receives a request for blocks it possesses, but it delays sending the blocks and by doing so, slows down the requester's block reception.
+As described in [the RFC](https://hyperledger.github.io/fabric-rfcs/text/0010-bft-signatures.html), block withholding resistance incurs a bandwidth and CPU cost,
+and therefore an operator of an orderer may opt out and use the standard mechanism by specifying `ReplicationPolicy`: `simple` in the local configuration.
```
Comment (denyeart) [inline release_notes/v3.0.0-beta.md:23]: I searched for ReplicationPolicy. It is not mentioned in the docs, and the orderer.yaml entry is not described well. It probably deserves a better description in orderer.yaml, and perhaps a mention in the docs, if that is a major part of the beta release.

Comment (yacovm) [reply release_notes/v3.0.0-beta.md:23]: i will do it, but, we can always update the docs post release, can't we?

Comment (yacovm) [reply release_notes/v3.0.0-beta.md:23]: I updated the `orderer.yaml`, i hope it's more clear now.

Comment (denyeart) [reply release_notes/v3.0.0-beta.md:23]: Yes, we can edit docs anytime. Just want to make sure anything we want to highlight in the beta is there. Merging.

**Review by denyeart — APPROVED** [review]

**Pull request merged**